### PR TITLE
[KOGITO-871] Prepare Operator 0.7.0-rc4 release

### PR DESCRIPTION
**Pull request by ricardozanini**
See:
https://issues.redhat.com/browse/KOGITO-871

This PR includes minor changes to prepare the `0.7.0-rc4` release in the dev preview channel.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -833,27 +833,66 @@ The output of this command is a ready-to-use Kogito Operator image that you can
...
 ### Deploying to OpenShift 4.x for development purposes
 
-To install the Kogito Operator on OpenShift 4.x for end-to-end (E2E) testing, ensure that you have access to a `quay.io` account to create an application repository. Follow the Operator Courier [authentication](https://github.com/operator-framework/operator-courier/#authentication) instructions to obtain an account token. This token is in the format `basic XXXXXXXXX` and both words are required for the command.
+To install the Kogito Operator on OpenShift 4.x for end-to-end (E2E) testing, ensure that you have access to a `quay.io` 
+account to create an application repository. 
 
-Push the Operator bundle to your quay application repository as shown in the following example:
+Follow the steps below:
+
+1. Run `make prepare-olm version=0.7.0`
```
Comment (radtriste) [inline README.md]: This does not work, need to be `make prepare-olm version=0.7.0-rc4` or ?

File: README.md [hunk 2]
```diff
@@ -833,27 +833,66 @@ The output of this command is a ready-to-use Kogito Operator image that you can
...
+3. Set courier variables:
 
-```bash
-## Kogito imagestreams should already be installed and available, for example:
-$ oc apply -f https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml -n openshift
+```
+$ export OPERATOR_DIR=build/_output/operatorhub/
+$ export QUAY_NAMESPACE=kiegroup # should be different in your environment
+$ export PACKAGE_NAME=kogito-operator
+$ export PACKAGE_VERSION=0.7.0
```
Comment (radtriste) [inline README.md]: Also here should be `0.7.0-rc4`

File: README.md (hunk 1 above)
Comment (radtriste) [reply README.md]: btw, testing this I get this error:
```
make[1]: Leaving directory '/home/tradisso/projects/kogito/kogito-cloud-operator'
ERROR: channel.currentCSV kogito-operator.v0.6.0 is not included in list of csvs [operatorhub/kogito-operator.package.yaml]
Resulting bundle is invalid, input yaml is improperly defined.
make: *** [Makefile:99: prepare-olm] Error 1
```

Comment (ricardozanini) [reply README.md]: This is just a README that will be changed back to 0.7.0. But anyway, I’ll change it

**Comment by ricardozanini** [comment]
> Ok for me.
> Please open a JIRA to remove the error happening when running `make prepare-olm version=0.7.0-rc4`
> 
> ```
> make[1]: Leaving directory '/home/tradisso/projects/kogito/kogito-cloud-operator'
> ERROR: channel.currentCSV kogito-operator.v0.6.0 is not included in list of csvs [operatorhub/kogito-operator.package.yaml]
> Resulting bundle is invalid, input yaml is improperly defined.
> make: *** [Makefile:99: prepare-olm] Error 1
> ```
> 
> as it can be confusing...

We can't remove this error since it's from the operator-courier verification step. What happens is that it tries to validate a package that mentions multiple CSVs that we don't have. The best approach here will be copying the previous version mentioned in the "alpha/stable" channel as well:
https://issues.redhat.com/browse/KOGITO-886

**Pull request merged**